packages/preview-middleware-client/src/adp/quick-actions/fe-v4/table-quick-action-base.ts

```diff
@@ -24,17 +24,20 @@ export abstract class TableQuickActionDefinitionBase {
     constructor(
         public readonly type: string,
         protected readonly defaultTextKey: string,
-        protected readonly context: QuickActionContext
+        protected readonly context: QuickActionContext,
+        protected readonly skipVariantManagement?: boolean
```

Comment:
Maybe better to name this parameter like isSkipVariantManagementCheck or isIgnoreVariantManagement
because it is boolean flag, its name should not sound like command

Reply:
adapted `isSkipVariantManagementCheck `